app/models/user.rb

```diff
@@ -32,4 +32,8 @@ def early_career_teacher?
   def mentor?
     mentor_profile.present?
   end
+
+  def preferred_name
+    username && username.presence || full_name
```

Comment:
Is there a nicer way to do this in Ruby? In JavaScript I'd just write `username || full_name`, but an empty string isn't falsey in ruby

Reply:
`username&.presence || full_name` would be shorter I guess.

Reply:
The presence is not needed. username || full_name will work just fine

Reply:
`username || full_name` evaluated to an empty string when `username` is an empty string

i'll go with what @kicferk1 said, thanks!